core/core-dotspacemacs.el

```diff
@@ -129,6 +129,14 @@ several times cycle between the kill ring content.'")
 (defvar dotspacemacs-guide-key-delay 0.4
   "Guide-key delay in seconds.")
 
+(defvar dotspacemacs-which-key-delay 0.4
+  "Which-key delay in seconds.")
+
+;; TODO: Implement me
+;; Possible options should be: right bottom right-then-bottom
+(defvar dotspacemacs-which-key-position 'right-then-bottom
+  "Which-key position in seconds.")
```

Comment:
I didn't suggest 'right. I suggested 'bottom. I have no problem with 'right-then-bottom. I just don't think it's the right default, because it's a little surprising to have the buffer sometimes opening on the bottom and sometimes on the right. I'd rather people explicitly choose that option when they understand the behavior. 

I'll let @syl20bnr decide on this though.